src/app/code/community/FireGento/Logger/Model/Logstash.php

```diff
@@ -83,7 +83,7 @@ protected function buildJSONMessage($event, $enableBacktrace = false)
         $fields['HttpHost'] = (!Mage::app()->getRequest()->getHttpHost()) ? 'cli': Mage::app()->getRequest()->getHttpHost();
         $fields['LogFileName'] = $this->_logFileName;
         // Only add session fields if a session was already instantiated and logger should not start a new session
-        if (isset($_SESSION)) {
+        if (isset($_SESSION) && isset($_SESSION['core'])) {
```

Comment:
The `isset($_SESSION)` conjunct can be removed (see also my other comment).

Reply:
ditto